concent_api/requirements.lock

```diff
@@ -4,31 +4,29 @@ cbor2==3.0.4
 certifi==2018.1.18
 cffi==1.11.4
 chardet==3.0.4
-coincurve==6.0.0
-Django==1.11.9
+coincurve==7.1.0
+Django==1.11.10
```

Comment:
Not a review, just a quick remark:

Please only update the stuff that comes from newer golem-messages. `Django` (and probably other packages that changed here) are not dependencies of golem-messages and updating them is not a part of this task. If you really want to update them, it should be a separate commit (so that we can easily see it and revert it if a newer version is broken without reverting other changes).

To get new dependencies of golem-messages do this:
- Create a clean virtualenv
- Install stuff from `requirements.lock`
- Install the new `golem-messages` with the `--upgrade` option
- Replace `requirements.lock` with the output of `pip freeze`

At this point in the diff you can see the packages that were added or updated because they are dependencies of `golem-messages`. To see what was removed you need additional steps:

- Run `pipdeptree` and see which top-level packages (packages listed without any indentation) are not in `requirements.txt`.
- Remove those packages from `requirements.lock`.


BTW review would be easier if updating golem-messages and fixing stuff broken after the update was a separate pull request.

Reply:
Updated. Only stuff that comes from golem-messages are updated.

Reply:
OK. Please don't merge this until @bartoszbetka resolves the problem with pyellpitic update. Otherwise we won't be able to deploy code master to the cluster.